service-front/docker-compose.env

```diff
@@ -3,3 +3,5 @@ API_BASE_URI=http://api-web
 SIRIUS_BASE_URL=http://sirius-mock:4010 # http://host.docker.internal:8080 for real Sirius
 SIRIUS_PUBLIC_URL=http://localhost:8080
 APP_DEBUG=1
+EXPERIAN_CROSSCORE_BASE_URL=https://experian-crosscore-mock/decisionanalytics/crosscore/cc_ prod_route
+EXPERIAN_CROSSCORE_AUTH_URL=http://experian-crosscore-auth-mock/oauth2/experianone/v1/token
```

Comment:
A few things:
- These should use the ports configured in docker-compose (looks like 4011/4012 respectively)
- This should be in the API, not the front container
- There's a latent space in `cc_ prod_route`
- `/decisionanalytics/crosscore/cc_prod_route` doesn't resolve on the CrossCore mock (even with the space fixed)